policybot/pkg/config/args.go

```diff
@@ -66,6 +66,27 @@ type Nag struct {
 	Message string
 }
 
+type FlakeChaser struct {
```

Comment:
Can you model this like the other features? That is, putting an instance of this type into the Org struct, and one in the top-level Args, and in your case another one in the Repo struct. So that gives you a global default, a per-org, and per-repo.

Reply:
I looked a bit and think, not sure.

- some settings does not seem requires per org/repo customization. e.g the inactive days.
- for nags/autolabels, i don't see how they're customized on per repo level either. currently it's empty in `org` section of the config

for now, it seems to me that the customization we need for this plugin, is to list the repo that needs enforcement. we can just have a list of repo name (with "istio/istio" to be fully qualified), and the filtering out based on the repo matching criteria. that might be good enough?

Reply:
Yeah, that's probably enough.